### DISCUSSION TITLE
Possible to use authorize! with class instead of instance?

### DISCUSSION THREAD

**Discussion by bjensen** [palkan/action_policy#160] [Q&A] [answered]
We have some general functionality that is not on an instance so we would like to be able to say: 

```
authorize! User, to: :edit? 
```

I have looked in the documentation but haven't been able to find it so I've come here..

**Comment by palkan** [comment, marked as answer]
Yes, it's totally possible (that would try to use `UserPolicy`).

Updated the lookup chain docs to mention that explicitly: https://github.com/palkan/action_policy/commit/f216d2c5fd390fd3e745429ae77526a9c72ec3dc

**Reply by bjensen** [reply]
Great. Thanks 👍
